src/main/scala/za/co/absa/hyperdrive/trigger/api/rest/controllers/WorkflowController.scala

```diff
@@ -153,7 +157,56 @@ class WorkflowController @Inject()(workflowService: WorkflowService) {
   @PostMapping(path = Array("/workflow/import"))
   def importWorkflow(@RequestPart("file") file: MultipartFile): CompletableFuture[WorkflowJoined] = {
     val workflowImport = ObjectMapperSingleton.getObjectMapper.readValue(file.getBytes, classOf[WorkflowImportExportWrapper])
-    workflowService.importWorkflow(workflowImport).toJava.toCompletableFuture
+    workflowService.convertToWorkflowJoined(workflowImport).toJava.toCompletableFuture
+  }
+
+  @PostMapping(path = Array("/workflows/import"))
+  def importWorkflows(@RequestPart("file") file: MultipartFile): CompletableFuture[Seq[Project]] = {
```

Comment:
Returns 200 when incorrect file is posted

Reply:
Fixed